app/assets/javascripts/admin/addon/services/admin-sidebar-state-manager.js

```diff
@@ -52,21 +47,21 @@ export default class AdminSidebarStateManager extends Service {
     if (isAdminSidebarActive) {
       return this.#forceAdminSidebar();
     } else {
-      this.isForcingAdminSidebar = false;
+      this.sidebarState.isForcingSidebar = false;
```

Comment:
`alias` gives me the heebee-jeebees. Let's just be explicit and include the service.